apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -165,7 +165,8 @@ DropletAutocompleteParameterTooltipManager.prototype.updateParameterTooltip_ = f
   }
   var tooltipInfo = this.dropletTooltipManager.getDropletTooltip(functionName);
 
-  if (currentParameterIndex >= tooltipInfo.parameterInfos.length) {
+  if (currentParameterIndex >= tooltipInfo.parameterInfos.length &&
+      (tooltipInfo.parameterInfos.length > 0 || currentParameterIndex > 0)) {
```

Comment:
A bit tough to follow this logic, could potentially introduce some intermediate variables for something like:

```
var parameterExists = currentParameterIndex >= tooltipInfo.parameterInfos.length;
var isFirstParamOfNoParamFunction = tooltipInfo.parameterInfos.length === 0 && currentParameterIndex === 0;
var shouldShowTooltip = parameterExists || isFirstParamOfNoParamFunction;
if (!shouldShowTooltip) {
  return;
}
```